frontend/src/pages/search.js

```diff
@@ -0,0 +1,62 @@
+import Link from 'next/link';
+import { Form, Input, Button, Checkbox } from 'antd';
+
+const axios = require('axios');
+// https://developers.google.com/youtube/v3/docs/search/list -YT API
+
+function Search() {
+  return (
+    <div>
+      <p> Hello There </p>
+      <SearchBox></SearchBox>
+    </div>
+  );
+}
+
+const layout = {
+  labelCol: { span: 8 },
+  wrapperCol: { span: 16 },
+};
+const tailLayout = {
+  wrapperCol: { offset: 8, span: 16 },
+};
+
+function SearchBox() {
+  const onFinish = async (values) => {
+    let result = await axios({
+      method: 'get',
+      url: `https://youtube.googleapis.com/youtube/v3/search?part=snippet&maxResults=25&q=${values.name}&type=video&key=AIzaSyBcWEFvenf9oWCPuhMboUCeCvbt3kPbntU`,
```

Comment:
this key should go in a .env file, otherwise it looks great! i sent in the slack instructions (to Kush) on how to do that

Reply:
@marclewis8 now that kush has figured out the .env on the frontend, you can copy what he did :)